packages/orchestrator-ui-components/src/configuration/version.ts

```diff
@@ -1 +1 @@
-export const ORCHESTRATOR_UI_LIBRARY_VERSION = '1.38.0';
+export const ORCHESTRATOR_UI_LIBRARY_VERSION = '1.38.1';
```

Comment:
i do not see a change in package.json, how come this one is bumped?

Reply:
When doing an npm release Version Packages will overwrite it to the newest one.

Reply:
Actually no.

My idea is that during build this file will be generated and then the deploy will continue with the generated file so it will display the new version. We can probably gitignore it as well.

Reply:
It's from this PR. https://github.com/workfloworchestrator/orchestrator-ui-library/pull/1385